assets/js/modules/analytics/components/common/EnableUniversalAnalytics.js

```diff
@@ -165,7 +171,7 @@ export default function EnableUniversalAnalytics( {
 					checked={ isUAEnabled }
 					onClick={ onChange }
 					hideLabel={ false }
-					disabled={ ! hasModuleAccess }
+					disabled={ shouldEnableUAToggle }
```

Comment:
This looks like a mistake, as when `shouldEnableUAToggle` is true, the component will return `null`, so by making this change the dropdown will never be disabled.

I'm wondering what prompted you to make the change though - is there an issue with the existing check for `! hasModuleAccess` ?

Reply:
>This looks like a mistake, as when shouldEnableUAToggle is true, the component will return null, so by making this change the dropdown will never be disabled.

Good catch, this is correct.

>I'm wondering what prompted you to make the change though - is there an issue with the existing check for ! hasModuleAccess ?

When we change the condition to check the valid property id, the toggle appears; however, it is disabled since the user whose access has been removed doesn't have access to make changes to the toggle.

Reply:
> When we change the condition to check the valid property id, the toggle appears; however, it is disabled since the user whose access has been removed doesn't have access to make changes to the toggle.

Thanks for clarifying! Sounds like a fix is in order then, just not using `shouldEnableUAToggle` like this...

Reply:
The fix should be `! shouldEnableUAToggle`, the toggle is "ON" (checked); however, it will be disabled to "OFF" it.